### ISSUE TITLE
Accessibility:  [Choice group]: extra focus with navigating with down arrow key

### ISSUE THREAD

**Issue by dantist1122** [microsoft/fluentui#13826]
### Environment Information

- **Package version(s)**: 7.120.2
- **Browser and OS versions**:
Microsoft Edge: Version 83.0.478.56 (Official build) (64-bit)
Windows 10 Enterprise 2004, OS build 19041.329

### Describe the issue:

In ChoiceGroup component if you do not provide text property, Narrator will read it as a simple input, if you add ariaLabel for the option it will read, but produce extra click while navigating with down arrow key, if you add title property it will let narrator read correctly and do not produce extra click, but if hover it will show browser default tooltip, please see codepen bellow,
Tried adding aria-labelledBy but it also produce extra click

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/dantist1122/pen/GRovNQj
1. open codepan
2. start Narrator app
3. navigate to label "Pick one" 
4. click down arrow key to go through checkbox

#### Actual behavior:

it produce extra click while navigating with narrator and down arrow key
![choice group](https://user-images.githubusercontent.com/44105538/85888574-b05fb700-b79e-11ea-9a5d-e947fad937db.gif)

#### Expected behavior:

It should read arialabel value, but do not produce extra click while navigating

**Comment by tylerthedeveloper** [comment]
Im experiencing this as well.

I have a ChoiceGroup and fill the options with a list of IChoiceGroupOption.

When in scan mode, it goes back and forth between the radio button and the text, repeating the text.


```
<ChoiceGroup
                                options={
                                    [
                                        {
                                            ariaLabel: "label",
                                            id: "id",
                                            key: "key",
                                            text: "text"
                                        },
```

It reads the aria label supplied to the constructed object, and then reads the text, reading both, but should it not only read the aria label || text, but not both?

**Comment by smhigley** [comment]
I can't reproduce the issue originally raised, using Edge 91 and Narrator on Windows build 19043. I'd guess that whatever the original problem was, it was solved between in one of those updates.

@tylerthedeveloper for your example, reaching the radio and label text separately seems desired. You've put visible text on the page that is no longer labelling the radio, so of course Narrator should reach it separately; otherwise a Narrator user wouldn't be able to reach that text at all.

To your suggestion on the order of aria vs. visible text -- this is actually already how naming controls works. The radio itself is already named by the `ariaLabel`, which overrides the label (i.e. the options `text` prop). If you jump by form field or tab, you'll hear the `ariaLabel` text and not the visible label text. Leaving aside that having separate visible text and `aria-label` text seems like a really bad idea at face value, do you have an example of what you think the desired markup should be, and an example of where you would not want Narrator to read both the aria-label and visible label text?
